application-accelerator/creating-accelerators/transforms/open-rewrite-recipe.hbs.md

```diff
@@ -3,8 +3,15 @@
 The `OpenRewriteRecipe` transform allows you to apply any [Open Rewrite](https://docs.openrewrite.org/)
 **Recipe** to a set of files and gather the results.
 
->**Note** Currently, only [Java-related recipes](https://docs.openrewrite.org/reference/dependency-version-selectors/java/)
-are supported. The engine leverages version `7.21.3` of Open Rewrite and parses
+>**Note** Currently, only the following Open Rewrite Recipes are supported:
+>  * [Java recipes](https://docs.openrewrite.org/reference/recipes/java)
+>  * [Maven recipes](https://docs.openrewrite.org/reference/recipes/maven)
+>  * [XML recipes](https://docs.openrewrite.org/reference/recipes/xml)
+>  * [YAML recipes](https://docs.openrewrite.org/reference/recipes/yaml)
+>  * [JSON recipes](https://docs.openrewrite.org/reference/recipes/json)
+>  * [Properties recipes](https://docs.openrewrite.org/reference/recipes/properties)
+>
+> The engine leverages version `7.21.3` of Open Rewrite and parses
```

Comment:
The current version is `7.30.1` but not sure we need to be that specific here since version used will change over time

Reply:
I removed mention of Open Rewrite in this line (didn't think it added much info if no version was there) and kept the portion on parsing Java files using Java 11 grammar.